chain-vote/src/commitment.rs

```diff
@@ -33,6 +33,20 @@ impl CommitmentKey {
         (self.commit_with_random(m, &r), r)
     }
 
+    /// Return a commitment of a boolean value, and the used randomness, `r`, where the latter is computed
+    /// from a `Rng + CryptoRng`
+    pub(crate) fn commit_bool<R>(&self, m: &bool, rng: &mut R) -> (GroupElement, Scalar)
```

Comment:
I don't think a immutable reference to a bool makes much sense, but I thought clippy would catch that